Modules/Filtering/ImageFeature/include/itkLaplacianSharpeningImageFilter.hxx

```diff
@@ -19,12 +19,12 @@
 #define itkLaplacianSharpeningImageFilter_hxx
```

Comment:
It would be good if commit message mentioned `LaplacianSharpeningImageFilter`.

Reply:
Nice.